src/main/app/form/parser.ts

```diff
@@ -22,10 +22,10 @@ export const setupCheckboxParser: CheckboxParser =
   ([key, field]) => {
     if ((field as FormOptions)?.type === 'checkboxes') {
       field.parser = formData => {
-        const checkbox = formData[key] ?? [];
+        const checkbox: string | string[] = formData[key] ?? [];
         let checkboxValues;
         if ((field as FormOptions).values.length > 1) {
-          checkboxValues = checkbox.filter(Boolean);
+          checkboxValues = typeof checkbox === 'string' ? [checkbox] : checkbox.filter(Boolean);
```

Comment:
why do we need this logic?

Reply:
my understanding is checkboxes is always going to be array

Reply:
When a single value was checked it was causing errors as it sent through as a string rather than an array, it caused a bug which this change resolved

Reply:
Unless your updates to the checkbox logic yesterday rectified this, I saw in other places it didn't cause an issue so may have been the way I implemented checkboxes in the content.ts file

Reply:
Either way, this would prevent the bug from reappearing in future if others do what I did

Reply:
no I think value for checkbox is always going to be array of string
I can't think of a scenario where value will be string in one case and array in another
checkbox is multiple selection type always unlike radio button which is single select always